src/prpy/serialization.py

```diff
@@ -327,37 +344,66 @@ def deserialize_kinbody(env, data, name=None, anonymous=False, state=True):
         'Robot' if data['is_robot'] else 'KinBody',
         data['name']
     )
-
-    link_infos = [
-        deserialize_link_info(link_data['info']) \
-        for link_data in data['links']
-    ]
-    joint_infos = [
-        deserialize_joint_info(joint_data['info']) \
-        for joint_data in data['joints']
-    ]
-
-    if data['is_robot']:
-        # TODO: Also load sensors.
-        manipulator_infos = [
-            deserialize_manipulator_info(manipulator_data['info']) \
-            for manipulator_data in data['manipulators']
-        ]
-        sensor_infos = []
-
-        kinbody = RaveCreateRobot(env, '')
-        kinbody.Init(
-            link_infos, joint_infos,
-            manipulator_infos, sensor_infos,
-            data['uri']
-        )
+    
+    name_desired = name or data['name']
+    
+    if 'uri_only' in data and data['uri_only']:
+        
+        if data['is_robot']:
+            
+            if data['uri'].endswith('.robot.xml'):
+                kinbody = env.ReadRobotXMLFile('robots/barrettwam.robot.xml')
+                kinbody.SetName(name_desired)
+                env.Add(kinbody, anonymous)
+            else:
+                urdf,srdf = data['uri'].split()
+                m_urdf = openravepy.RaveCreateModule(env, 'urdf')
```

Comment:
Is there any way we can create _one_ `or_urdf` module for the entire de-serialization, instead of creating a new one for each robot?